wiki/misc/worldtypes.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Adding a Custom World Preset
```

Comment:
after rereading a few times, the tone of this article is a bit off imo. it offers extremely short explanations of each thing (often without actually explaining anything. "Configurable noise generator." tells most people absolutely nothing), tosses pretty much all its information into one block in the middle instead of spacing it out across multiple code snippets like other wiki articles, and via those two vectors just has a generally terse tone, like it's pissed off at the user for not knowing about this and having to read the article

sorry to be mean, but I feel like this needs to be rewritten instead of getting bandaid fixes for review comments